src/fitnesse/reporting/SuiteHtmlFormatter.java

```diff
@@ -31,13 +42,41 @@ public class SuiteHtmlFormatter extends InteractiveFormatter implements Closeabl
   private String testSummariesId = TEST_SUMMARIES_ID;
   private boolean testSummariesPresent;
   private TimeMeasurement totalTimeMeasurement;
+  private LinkedList<String> instructionsHtmlText;
+  private int testSystemListenerShowInstructions;
+  private boolean testSystemListenerShowHtmlTable;
+  private int testSystemListenerSleep;
 
 
   public SuiteHtmlFormatter(WikiPage page, boolean testSummariesPresent, Writer writer) {
     super(page, writer);
     this.testSummariesPresent = testSummariesPresent;
     totalTimeMeasurement = new TimeMeasurement().start();
     testBasePathName = PathParser.render(page.getFullPath());
+    String sbys = getPage().getVariable(SLIM_SHOW);
+    testSystemListenerShowHtmlTable = (sbys != null);
+    testSystemListenerShowInstructions = 0;
+    if(testSystemListenerShowHtmlTable){
```

Comment:
```suggestion
    if (testSystemListenerShowHtmlTable) {
```

Missing whitespaces like this in many places